packages/calcite-components/src/components/tab-title/tab-title.scss

```diff
@@ -292,4 +292,8 @@
   :host([bordered][position="bottom"][selected]) .container {
     border-block-start-style: none;
   }
+
+  .close-button {
+    @apply z-default;
```

Comment:
Could you add a comment explaining why the z-index is necessary in high contrast mode? 

@SkyeSeitz @ashetland Would `tab-title`'s close button end up at some point being circular (similar to `chip`)? Currently, the focus outlines from the title's container and close button mostly overlap and ended up causing some trouble in high contrast mode.

Reply:
Yep, I think `tab-title` is a good candidate for the round button. We spec'd it out like the current design to match behaviors elsewhere until we could look at all these more holistically. #6640

Reply:
There might be padding tweaks needed to accommodate a round button.

Reply:
I've added the comment @jcfranco